app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -256,6 +259,7 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
     debug('Checking if archive exists ...');
     const jobPromise = fetchArchivingJobForAgenda(
       this.args.currentAgenda,
+      this.selectedMandatees,
       this.store
     );
     const [name, job] = await all([namePromise, jobPromise]);
```

Comment:
Saw this in next PR, the `namePromise` is always the same.
Might cause file overwrites on local system if someone decided to download all documents 1 mandatee at a time.